platform/testbed/src/integrationTest/resources/api/func/deploy_not_found/01.lua

```diff
@@ -0,0 +1,33 @@
+sys = require('sys')
+
+req = fail_on_error(sys.func.create({
+    name = 'test-func',
+    inputs = {},
+    code = [[4 + 2]]
+
+}))
+
+sys.await_completed(req)
+
+func = fail_on_error(sys.func.get(req.id))
+assert(func.code_deployed == 1)
+
+err1, res1 = sys.func.deploy({
+    id = func.id,
+    version = 3
+})
+
+assert(err1.message == 'Code not found')
+assert(err1['message'] == 'Code not found')
+assert(res1 == nil)
+
+err2, res2 = sys.func.deploy({
```

Comment:
```suggestion
err, res = sys.func.deploy({
```

just reuse those variable